### PR TITLE
Update resources in walkthrough

### PR DESCRIPTION
**Pull request by pmorie**
I noticed earlier while preparing a talk that the some of the resources in the walkthrough were out of date.

### CODE REVIEW

**Comment by duglin** [comment]
LGTM

File: docs/walkthrough.md [hunk 1]
```diff
@@ -123,19 +129,69 @@ We should see something like:
...
+
+We should see something like:
+
+```console
+NAME                                   EXTERNAL NAME
+86064792-7ea2-467b-af93-ac9694d96d52   default
+```
+
+**NOTE:** Just like in the command above, we used a custom set of columns.
+The `NAME` field is the Kubernetes name of the ClusterServiceClass and the
```
Comment (kibbles-n-bytes) [inline docs/walkthrough.md]: `ClusterServiceClass` -> `ClusterServicePlan`, correct?

Comment (pmorie) [reply docs/walkthrough.md]: Yerp, nice catch

**Comment by pmorie** [comment]
@kibbles-n-bytes addressed, PTAL

**Comment by kibbles-n-bytes** [comment]
@pmorie It seems like the spot I was referencing still says `ClusterServiceClass`

**Pull request merged**